pages/login.js

```diff
@@ -0,0 +1,3 @@
+export default function produto() {
```

Comment:
alterar nome da função

```suggestion
export default function login() {
```